docs/randomblind.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Randomblind issuance
+---
+
+<script type="text/x-mathjax-config">
+  MathJax.Hub.Config({
+    extensions: ["tex2jax.js"],
+    jax: ["input/TeX", "output/HTML-CSS"],
+    tex2jax: {
+      inlineMath: [ ['$','$'], ["\\(","\\)"] ],
+      displayMath: [ ['$$','$$'], ["\\[","\\]"] ],
+      processEscapes: true
+    },
+    "HTML-CSS": { fonts: ["TeX"] }
+  });
+</script>
+<script type="text/javascript" async src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js"></script>
+
+*Randomblind* issuance is a new IRMA feature introduced in
+[irmago v0.6.0](https://github.com/privacybydesign/irmago/releases/tag/v0.6.0)
+that can give the user extra privacy
+guarantees during issuance.  Specifically, if this mode of issuance is enabled
+for an attribute within a credential type, this attribute is guaranteed to 
+  1) be random and 
```

Comment:
Suggestion: clarify by adding:`i.e., unequal to all previously issued instances of this attribute`